main.go

```diff
@@ -217,6 +217,8 @@ func main() {
 
 		// dotnet sonarscanner begin /k:"Travix.Core.ShoppingCart" /d:sonar.host.url=https://sonarqube.travix.com /d:sonar.cs.opencover.reportsPaths="**\coverage.opencover.xml" /d:sonar.coverage.exclusions="**Tests.cs"
```

Comment:
Please update the comment as well with the exact command

Reply:
Hey @snigdhasambitak, this at the moment is just a temporary experiment, is it okay if we leave it like this for the time being?

Reply:
Yeah, sure. I thout we have verified this locally before this change. Can leave it as it is for the moment